frontend/src/components/organism/Projections.tsx

```diff
@@ -0,0 +1,56 @@
+import moment from 'moment';
+import React from 'react';
+import styled from 'styled-components';
+import { useTransactionState } from '../../store/contexts/transactions';
+
+const Projections: React.FC<{ className?: string }> = ({ className }) => {
+  const transactionState = useTransactionState();
```

Comment:
Here, if you just want the `transactions` key, you could write it as

```typescript
const { transactions } = useTransactionState();
```

Bit easier on the eyes IMO, but "smak og behag".